bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -761,7 +761,7 @@ class BCStateTran : public IStateTransfer {
   struct SourceSession {
    public:
     SourceSession(logging::Logger& logger, uint64_t sourceSessionExpiryDurationMs)
-        : logger_(logger), expiryDurationMs_{sourceSessionExpiryDurationMs}, startTime_{0} {}
+        : logger_(logger), expiryDurationMs_{sourceSessionExpiryDurationMs}, replicaId_{0}, startTime_{0} {}
```

Comment:
nit:
All native types dont need zero in brackets. just use {}. Same  commentfor all other places below.

Reply:
I followed the convention as I saw other native types being explicitly initialized. I've decided to keep it this way because it also gives information about the type of the variable.